specification.rst

```diff
@@ -179,9 +179,10 @@ In independent styles, ``cs:info`` has the following child elements:
     indicates the field(s) for which the style is relevant.
 
 ``cs:id``
-    Must appear once. The element should contain a URI to establish the identity
-    of the style. A stable, unique and dereferenceable URI is desired for
-    publicly available styles.
+    Must appear once, and contain a URI to establish the identity
+    of the style. For publicly available styles, this URI must be stable and 
```

Comment:
"must" is strong language in a spec, and probably not appropriate here; "should" could be coupled with a following sentence that explains why?